### PR TITLE
CUDA: fix autoupdate

### PR DESCRIPTION
**Pull request by linsui**
https://github.com/ScoopInstaller/Main/issues/150

Wondering why hash extract not work without au.hash.regex.

### CODE REVIEW

File: bucket/cuda.json [hunk 1]
```diff
@@ -35,11 +35,12 @@
     "autoupdate": {
         "architecture": {
             "64bit": {
-                "url": "https://developer.nvidia.com/compute/cuda/$majorVersion.$minorVersion/Prod/local_installers/cuda_$version_win10.exe#/dl.7z"
+                "url": "https://developer.download.nvidia.com/compute/cuda/$majorVersion.$minorVersion/Prod/local_installers/cuda_$version_win10.exe#/dl.7z"
             }
         },
         "hash": {
-            "url": "https://developer.download.nvidia.com/compute/cuda/$majorVersion.$minorVersion/Prod/docs2/sidebar/md5sum.txt"
+            "url": "https://developer.nvidia.com/cuda-downloads?target_os=Windows&target_arch=x86_64&target_version=10&target_type=exelocal",
```
Comment (Ash258) [inline bucket/cuda.json]: What about this?

http://developer.download.nvidia.com/compute/cuda/10.1/Prod/docs/sidebar/md5sum.txt

A

Comment (linsui) [reply bucket/cuda.json]: There is no hash for this version. And the md5 file url is changing...(docs2 ->docs)

**Comment by Ash258** [comment]
Also sidenote. What is difference between `win10` and `windows` installers?

**Comment by linsui** [comment]
`win10` for win10 and windows server 2016/2019, `windows` for older windows version.

**Comment by Ash258** [comment]
Obviously, but what is the actual reason in that binaries?

**Comment by linsui** [comment]
Do not know... I guess it's related to drivers.

**Pull request merged**